krun.py

```diff
@@ -252,13 +277,17 @@ def run(self):
 
             # We dump the json after each experiment so we can monitor the
             # json file mid-run. It is overwritten each time.
-            dump_results(self.config_file, self.out_file, self.results,
-                      self.platform.audit)
+            util.dump_results(self.config_file, self.out_file, self.results,
+                              self.platform.audit)
 
             self.jobs_done += 1
             self.platform.wait_until_cpu_cool()
             self.platform.check_dmesg_for_changes()
 
+            # FIX BEFORE MERGE
```

Comment:
Grepping through the code, I see that this is the only ENV VAR we get. I think now we have a command line parser it would be sensible (and easier for documentation, etc.) to make this a command-line argument.